src/content/function-info.js

```diff
@@ -213,3 +213,35 @@ export function getFunctionInfo(fullName) {
          getJSFunctionInfo(fullName) ||
          getFallbackFunctionInfo(fullName);
 }
+
+/**
+ * Strip any function arguments from the given string.
+ *
+ * If the function fails to determine that there are any parentheses to strip
+ * it will return the original string.
+ */
+export function stripFunctionArguments(functionCall) {
+  // Remove known data that can appear at the end of the string
+  const s = functionCall.replace(/ \[clone [^]+\]$/, '').replace(/ const$/, '');
+
+  // Start from the end of the string and assume we are starting on a
+  // right parenthesis.  Then iterate towards the beginning until we
+  // find the matching left parenthesis.
+  let i = s.length - 1;
```

Comment:
I used `i` outside the for loop. But now I changed it to use an early return, so I can move `i` into the initialization part.